llvm/test/CodeGen/SystemZ/int-abs-01.ll

```diff
@@ -85,8 +93,13 @@ define i64 @f7(i64 %val) {
 ; Test another form of f6, which is that produced by InstCombine.
 define i64 @f8(i64 %val) {
 ; CHECK-LABEL: f8:
-; CHECK: lpgfr %r2, %r2
-; CHECK: br %r14
+; CHECK:       # %bb.0:
+; CHECK-NEXT:    sllg %r0, %r2, 32
+; CHECK-NEXT:    srag %r2, %r0, 32
+; CHECK-NEXT:    cgibhe %r0, 0, 0(%r14)
```

Comment:
All the actual changes in the SystemZ tests look to be clear regressions.  It doesn't appear to recognize the "abs" patterns with sign extension and more.